Resources/doc/versioning.rst

```diff
@@ -0,0 +1,180 @@
+API versioning
+==============
+Why version your API ?
```

Comment:
please add a new line before and after each headline (before line 3 and after line 4 in this example)